examples/postgres/vagga.yaml

```diff
@@ -6,31 +6,76 @@ containers:
   ubuntu:
     setup:
     - !Ubuntu xenial
-    - !Install
-      - postgresql-9.5
+    # Use fixed user id and group id for postgres, because in some cases
+    # we may need to upgrade (rebuild) a postgres container, but keep the data
+    # on a `!Persistent` volume still usable. User ids in ubuntu packages are
+    # not guaranteed to be same on every installation.
+    #
+    # The command-lines are from the postgres-common package except
+    # added --uid 200 --gid 200
+    - !Sh |
+        addgroup --system --gid 200 postgres
```

Comment:
Basically, we add commands for one of the three reasons:

1. If expressing it as a shell command is too long and hard to remember
2. If we can make much better versioning of this thing if it's written as a built-in command
3. If it's impossible to do using command-line

I think neither one is applied to this command. These ones are well known and well documented. We aren't going to reimplement every single command-line utility :)